### PR TITLE
Feature/add admin booking info endpoint

### PR DESCRIPTION
**Pull request by DFSko**

### CODE REVIEW

File: Api/Controllers/AdministratorControllers/BookingsController.cs [hunk 1]
```diff
@@ -84,6 +88,27 @@ public async Task<IActionResult> GetAgentBookings([FromRoute] int agentId)
 
             return Ok(bookings);
         }
+        
+        
+        /// <summary>
+        ///     Gets booking data by reference code.
+        /// </summary>
+        /// <param name="referenceCode">Booking reference code</param>
+        /// <returns>List of bookings</returns>
```
Comment (mingazhev) [inline Api/Controllers/AdministratorControllers/BookingsController.cs:97]: does it return a list?

Comment (DFSko) [reply Api/Controllers/AdministratorControllers/BookingsController.cs]: Fixed

File: Api/Controllers/AdministratorControllers/BookingsController.cs [hunk 2]
```diff
@@ -84,6 +88,27 @@ public async Task<IActionResult> GetAgentBookings([FromRoute] int agentId)
...
+        [HttpGet("bookings/{referenceCode}")]
+        [ProducesResponseType(typeof(AccommodationBookingInfo), (int) HttpStatusCode.OK)]
+        [ProducesResponseType(typeof(ProblemDetails), (int) HttpStatusCode.BadRequest)]
+        [AdministratorPermissions(AdministratorPermissions.AgentManagement)]
```
[the remark this answers is not in the document]
Comment (DFSko) [reply Api/Controllers/AdministratorControllers/BookingsController.cs:101]: We have BookingManagement permission, but all other endpoints in this controller usings AgentManagement

**Review by mingazhev — APPROVED** [review]

File: Api/Controllers/AdministratorControllers/BookingsController.cs (hunk 2 above)
[the remark this answers is not in the document]
Comment (DFSko) [reply Api/Controllers/AdministratorControllers/BookingsController.cs]: Fixed

**Pull request merged**